about.html

```diff
@@ -4,9 +4,107 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="style.css">
+  <title>Online Course development web about page</title>
 </head>
 <body>
-  
+  <header>
+    <div class="page-media">
+      <div class="page-media-container">
+        <ul>
+          <li><a href="#"><img src="./assets/images/facebook-2-logo-svgrepo-com.svg" alt="facebook-logo" width="20rem"></a></li>
+          <li><a href="#"><img src="./assets/images/twitter.svg" alt="twitter-logo" width="20"></a></li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">MyPage</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
+    </div>
+    <div class="header-navigator">
+      <a id="logo" href="./index.html"><img src="./assets/images/online-learning-course-home-schooling-education-and-graduation-logo-free-vector.webp" alt="online course logo"  width="100"></a>
+      <nav id="navigator"><ul>
+        <li><a href="./index.html">Home</a></li>
+        <li><a href="#about-logo">Logo</a></li>
+        <li><a href="#past-courses">Course</a></li>
+        <li><a href="#sponsor">Sponsor</a></li>
+      </ul></nav>
+    </div>
+    <nav id="hamburger-button">
+      <div class="bar1"></div>
+    <div class="bar2"></div>
+    <div class="bar3"></div>
+    </nav>
+      <div class="mobile-menu-modal toogleModalDisplay">
+      <div class="mobile-menu">
+        <ul>
+         <li><a href="./index.html">Home</a></li>
+          <div class="mobile-menu-separator"></div>
+            <li><a href="#about-logo">Logo</a></li>
+          <div class="mobile-menu-separator"></div>
+          <li><a href="#past-courses">Course</a></li>
+           <div class="mobile-menu-separator"></div>
+           <li><a href="#sponsor">Sponsor</a></li>
+           <div class="mobile-menu-separator"></div>
+        </ul>
+      </div>
+    </div>
+  </header>
+      <section class="about-headline">
+      <div class="introduction introduction-about">
+         <h1 class="heading-title"> Web development course For 2023</h1>
+         <p class="description about-description">Online Course’s mission is to provide flexible, effective skill development to empower organizations and individuals. <br>
+          Online Course’ Business helps companies achieve critical business outcomes and stay competitiv.e by offering engaging on-demand, immersive, and cohort-based learning.
+        </p>
+         <p class="contact"> Please Contact us per Email for any further questions about Online Course! <br> <a href="">online@course.org</a>  
+         </p>
+      </div>
+    </section>
+    <section id="about-logo">
+      <p class="logo-title">Online Course 2023 logo</p>
```

Comment:
- [ ] I think here you should use the `hx` tag for best practice as it's a title 😉